ide-common/src/main/java/org/digma/intellij/plugin/analytics/Environment.java

```diff
@@ -115,6 +116,7 @@ public List<String> getEnvironments() {
     public void refreshNowOnBackground() {
 
         Log.log(LOGGER::trace, "Refreshing Environments on background thread.");
+        Log.test(LOGGER::info, "Firing Refreshing Environments");
         Backgroundable.ensureBackground(project, "Refreshing Environments", () -> {
```

Comment:
info will explode the user log.
same for all other calls to Log.test , i'm not going to write more comments about it.